[DOCS] Adds 8.2 What's new (#130342)
elastic/kibana: docs/user/whats-new.asciidoc

coming::[{minor-version}]

Here are the highlights of what's new and improved in {minor-version}.
For detailed information about this release,
check the <<release-notes, release notes>>.

Previous versions: {kibana-ref-all}/8.1/whats-new.html[8.1] | {kibana-ref-all}/8.0/whats-new.html[8.0]

//NOTE: The notable-highlights tagged regions are re-used in the
//Installation and Upgrade Guide

// tag::notable-highlights[]

[float]
=== Discover

[float]
==== Document Explorer on by default

*Document Explorer*, the powerful new way to analyze and explore your data with *Discover*,
is now on by default. *Document Explorer* helps you quickly sort, select, and compare data,
resize columns, and view documents in fullscreen. For more information,
check {kibana-ref}/document-explorer.html[Document Explorer].

[role="screenshot"]
image::images/highlights-discover.png[A view of the the Document Explorer in Discover]

[float]
=== Visualization editors

[float]
==== Auto-apply your changes in Lens

By default, *Lens* has always automatically applied the changes you make to visualizations.
Now you can control when *Lens* applies your changes and generates suggestions with

> he new ability to turn on and off auto-apply. To start manually applying your changes,

Reviewer: ```suggestion
the new ability to turn on and off auto-apply. To start manually applying your changes,
```

[role="screenshot"]
image::images/highlights-lens-annotations.png[Annotations in Lens visualzations]

[float]
==== Open TSVB visualizations in Lens

Open and edit your Time Series *TSVB* visualizations in *Lens*,
the drag-and-drop visualization editor that provides you with additional
visualization types, reference lines, and more.
For more information, check {kibana-ref}/tsvb.html#edit-visualizations-in-lens[Edit visualizations in Lens].

[role="screenshot"]
image::images/highlights-lens-tsvb.gif[TSVB visualization in Lens]

[float]
==== Explore Lens visualization data in Discover

For further exploration of your data, you can now open *Lens* visualization data
in *Discover*. With *Discover* you can view the documents in the fields you used
to create the visualization, then navigate back to *Lens* to finish creating your
visualization. For more information,
check {kibana-ref}/lens.html#explore-lens-data-in-discover[Explore the data in Discover].

[role="screenshot"]
image::images/highlights-lens-discover.gif[Seamlessly move from Lens to Discover to further explore your data]

[float]
==== Enhancements to visualization editors

> {kib} offers even more ways to create visualizations:

Reviewer: ```suggestion
{kib} offers even more ways to create visualizations:

```

* **Metric visualization.** Use the new Metric visualization type in **Lens**
to show the latest or earliest data, or the last value.
* **Group by multiple fields in TSVB.** When you select **Terms** from the **Group by**

> dropdown in ****TSVB, you can calculate the top values for multiple fields.

Reviewer: ```suggestion
dropdown in **TSVB**, you can calculate the top values for multiple fields.
```

* **Customize the legend size.** In all visualization editors,
you can now increase or decrease the visibility of your legend by changing the size.

[float]
=== Maps

[float]
==== Hexagons

In *Maps*, you can now display your clusters as hexagons
instead of rectangles. This feature adds another beautiful, and more precise way, to
visualize your geo_point fields. Hexagon aggregation is
a https://www.elastic.co/subscriptions[subscription feature].

[role="screenshot"]
image::images/highlights-maps-hexagon.png[region map created in Lens]

[float]
==== Region Maps in Lens

You can now create region maps

> (also called cloropeth maps) from *Lens*.

Author: ```suggestion
(also called choropleth maps) from *Lens*.
```

[role="screenshot"]
image::images/highlights-maps-region.png[Apply changes to render visualization]

[float]
==== Custom icons

Upload an SVG image and use it as icons in your maps.

> Edit your maps layer,and then select

Reviewer: ```suggestion
Edit your maps layer, and then select
```

unexpectedly or during known downtimes.

[role="screenshot"]
image::images/highlights-alerting-snooze.png["UI for enabling snooze mode", width=50%]

[float]
==== Rule history

For better observability and monitoring of your rules, {kib} now
displays the execution history of your rules and actions. You can customize the
history view to include granular information with an array of impactful data points,
including the underlying alerting query durations,
the number of actions that were triggered or errored, alert context,
rule execution timeouts, and more.

[role="screenshot"]
image::images/highlights-alerts-rule-history.png[Rule history table in Alerting]

[float]
==== Configure rules

*Alerting* introduces configurable guardrails so that as an admin,
you can apply your policies on a level deeper than RBAC and fully stay on top of
the stress that the alerting system may put on your instance in case your users misconfigured rules.
This way you have peace of mind that your instance remains performant.

Specifically, in the kibana.yml file:

* Configure your rule queries to timeout after a custom time expires without the query returning.

> You can do this on a per rule type basis too. Refer to {kibana-ref}/alert-action-settings-kb.html[documentation] for the `rules.execution.timeout setting.``

Reviewer: ```suggestion
You can do this on a per rule type basis too. Refer to {kibana-ref}/alert-action-settings-kb.html[documentation] for the `rules.execution.timeout` setting.
```

in *Discover*. With *Discover* you can view the documents in the fields you used
to create the visualization, then navigate back to *Lens* to finish creating your
visualization. For more information,
check {kibana-ref}/lens.html#explore-lens-data-in-discover[Explore the data in Discover].

[role="screenshot"]
image::images/highlights-lens-discover.gif[Seamlessly move from Lens to Discover to further explore your data]

[float]
==== Enhancements to visualization editors

{kib} offers even more ways to create visualizations:

* **Metric visualization.** Use the new Metric visualization type in **Lens**
to show the latest or earliest data, or the last value.

* **Group by multiple fields in TSVB.** When you select **Terms** from the **Group by**
dropdown in **TSVB**, you can calculate the top values for multiple fields.

* **Customize the legend size.** In all visualization editors,
you can now increase or decrease the visibility of your legend by changing the size.

[float]
=== Maps

[float]
==== Display clusters as hexagons

In *Maps*, you can now display your clusters as hexagons
instead of rectangles. This feature adds another beautiful, and more precise way, to

> visualize your geo_point fields. Hexagon aggregation is

Author: ```suggestion
visualize your geo_point fields. Hexagon aggregations require
```

> a basic license on Cloud and a platinum license on prem.

Reviewer 2: ```suggestion
a standard subscription on Cloud and a platinum license on prem.
```

I'm not 100% sure that standard is correct, it might be gold or platinum.

Reviewer 2: I'm checking with Cloud PM to confirm.

Reviewer 3: Fly-by comment, but this is likely the kind of detail that doesn't belong in the docs anyway.

[role="screenshot"]
image::images/highlights-maps-custom-icon-2.png[custom phone icon added to a map]

[float]
=== Alerting

[float]
==== Temporarily pause rules

*Alerting* introduces the ability to snooze rules to suppress notifications
and actions for a user-defined duration. You'll never miss an
action because you forgot to unmute a rule while handling problems that arose
unexpectedly or during known downtimes.

[role="screenshot"]
image::images/highlights-alerting-snooze.png["UI for enabling snooze mode", width=50%]

[float]
==== View rule history

For better observability and monitoring of your rules, {kib} now
displays the execution history of your rules and actions. You can customize the
history view to include granular information with an array of impactful data points,
including the underlying alerting query durations,
the number of actions that were triggered or errored, alert context,
rule execution timeouts, and more.

[role="screenshot"]
image::images/highlights-alerts-rule-history.png[Rule history table in Alerting]

[float]
==== Configure rules

*Alerting* introduces configurable guardrails so that as an admin,

> you can apply your policies on a level deeper than RBAC and fully stay on top of

Reviewer 3: ```suggestion
you can apply your policies on a level deeper than role-based access control (RBAC) and fully stay on top of
```

the stress that the alerting system may put on your instance, in case your users misconfigured rules.
This way you have peace of mind that your instance remains performant.

Specifically, in the kibana.yml file:

* Configure your rule queries to timeout after a custom time expires without the query returning.

> You can also do this on a per rule type basis.

Reviewer 3: ```suggestion
You can also do this on a per rule type basis with the `xpack.alerting.run.timeout` setting.
```

> Check the {kibana-ref}/alert-action-settings-kb.html#action-settings[documentation] for the `xpack.alerting.run.timeout` setting.

Reviewer 3: ```suggestion

```

> * Configure a minimum value for the interval with which your rules run. Strictly enforce the

Reviewer 3: ```suggestion
* Configure a minimum value for the interval with which your rules run with the `xpack.alerting.rules.minimumScheduleInterval.value` setting. Strictly enforce the
```

> minimum interval or just display warnings on the UI. Check the

Reviewer 3: ```suggestion
minimum interval or just display warnings on the UI with the `xpack.alerting.rules.minimumScheduleInterval.enforce` setting.
```

>  {kibana-ref}/alert-action-settings-kb.html#action-settings[documentation] for the

Reviewer 3: ```suggestion

```

> `xpack.alerting.rules.minimumScheduleInterval` settings.

Reviewer 3: ```suggestion

```

> * Configure the maximum number of actions that a rule can generate. Specify a different cap for your

Reviewer 3: ```suggestion
* Configure the maximum number of actions that a rule can generate with the `xpack.alerting.rules.run.actions.max` setting. Specify a different cap for your
```

> different rule types, depending on your needs.

Reviewer 3: ```suggestion
different rule types with the `xpack.alerting.rules.run.ruleTypeOverrides` setting, depending on your needs.
```

> Check the {kibana-ref}/alert-action-settings-kb.html#action-settings[documentation] for the `xpack.alerting.run.actions.max` setting.

Reviewer 3: ```suggestion

```

> 

Reviewer 3: Since all three links above go to the same place, IMO it makes sense to just put the link once at the bottom:
```suggestion
For more information, refer to {kibana-ref}/alert-action-settings-kb.html#action-settings[Action settings].
```

[float]
==== New xMatters connector

> With the new xMatters connector, you can

Reviewer 3: ```suggestion
With the new <<xmatters-action-type,xMatters connector>>, you can
```

is now on by default. *Document Explorer* helps you quickly sort, select, and compare data,
resize columns, and view documents in fullscreen. For more information,
check {kibana-ref}/document-explorer.html[Explore your documents].

[role="screenshot"]
image::images/highlights-discover.png[A view of the the Document Explorer in Discover]

[float]
=== Visualization editors

[float]
==== Auto-apply your changes in Lens

By default, *Lens* has always automatically applied the changes you make to visualizations.
Now you can control when *Lens* applies your changes and generates suggestions with
the new ability to turn on and off auto-apply. To start manually applying your changes,
click *Settings* in the toolbar, then deselect *Auto-apply visualization changes*.
For more information, check {kibana-ref}/lens.html#create-the-visualization-panel[Create visualizations].

[role="screenshot"]
image::images/highlights-lens-auto-apply.png[Auto-apply changes in Lens]

[float]
==== Add annotations to Lens visualizations (technical preview)

To call out significant changes and trends in your data, you can now add annotations

> to *Lens* visualizations in tech preview. Customize the appearance of annotations with text,

Author: ```suggestion
to *Lens* visualizations. Customize the appearance of annotations with text,
```

[role="screenshot"]
image::images/highlights-lens-annotations.png[Annotations in Lens visualzations]

[float]
==== Open TSVB visualizations in Lens

Open and edit your Time Series *TSVB* visualizations in *Lens*,
the drag-and-drop visualization editor that provides you with additional
visualization types, reference lines, and more.
For more information, check {kibana-ref}/tsvb.html#edit-visualizations-in-lens[Edit visualizations in Lens].

[role="screenshot"]
image::images/highlights-lens-tsvb.gif[TSVB visualization in Lens]

[float]
==== Explore Lens visualization data in Discover

For further exploration of your data, you can now open *Lens* visualization data
in *Discover*. With *Discover* you can view the documents in the fields you used
to create the visualization, then navigate back to *Lens* to finish creating your
visualization. For more information,
check {kibana-ref}/lens.html#explore-lens-data-in-discover[Explore the data in Discover].

[role="screenshot"]
image::images/highlights-lens-discover.gif[Seamlessly move from Lens to Discover to further explore your data]

[float]
==== Enhancements to visualization editors

> {kib} offers even more ways to create visualizations:

Author: ```suggestion
{kib} now offers even more ways to create visualizations:
```

(RBAC) and fully stay on top of the stress that the alerting system may put on
your instance, in case your users misconfigured rules. This way you have peace
of mind that your instance remains performant.

Specifically, in the `kibana.yml` file:

* Configure your rule queries to timeout after a custom time expires without the
query returning. You can also do this on a per rule type basis with the
`xpack.alerting.run.timeout` setting.
* Configure a minimum value for the interval with which your rules run with the
`xpack.alerting.rules.minimumScheduleInterval.value` setting. Strictly enforce
the minimum interval or just display warnings on the UI with the
`xpack.alerting.rules.minimumScheduleInterval.enforce` setting.

* Configure the maximum number of actions that a rule can generate with the
`xpack.alerting.rules.run.actions.max` setting. Specify a different cap for your
different rule types with the `xpack.alerting.rules.run.ruleTypeOverrides`
setting, depending on your needs.

For more information, refer to
{kibana-ref}/alert-action-settings-kb.html#action-settings[Action settings].

[float]
==== New xMatters connector

> With the new <<xmatters-action-type,xMatters connector>>, you can

Author: ```suggestion
With the new {kibana-ref}/xmatters-action-type.html[xMatters connector], you can
```

in *Discover*. With *Discover* you can view the documents in the fields you used
to create the visualization, then navigate back to *Lens* to finish creating your
visualization. For more information,
check {kibana-ref}/lens.html#explore-lens-data-in-discover[Explore the data in Discover].

[role="screenshot"]
image::images/highlights-lens-discover.gif[Seamlessly move from Lens to Discover to further explore your data]

[float]
==== Enhancements to visualization editors

{kib} offers even more ways to create visualizations:

* **Metric visualization.** Use the new Metric visualization type in **Lens**
to show the latest or earliest data, or the last value.

* **Group by multiple fields in TSVB.** When you select **Terms** from the **Group by**
dropdown in **TSVB**, you can calculate the top values for multiple fields.

* **Customize the legend size.** In all visualization editors,
you can now increase or decrease the visibility of your legend by changing the size.

[float]
=== Maps

[float]
==== Display clusters as hexagons

In *Maps*, you can now display your clusters as hexagons
instead of rectangles. This feature adds another beautiful, and more precise way, to

> visualize your geo_point fields. Hexagon aggregations require

Author: ```suggestion
visualize your geo_point fields. 
```

> a standard subscription on Cloud and a platinum license on prem.

Author: ```suggestion

```
